metadata/update_metadata_features.py

```diff
@@ -0,0 +1,41 @@
+import argparse
+import json
+import os
+import pandas as pd
+
+
+def main(feature: str, domain: str, feature_dir: str, metadata_file: str):
+    metadata = json.load(metadata_file)
+    for filename in os.listdir(feature_dir):
+        f = os.path.join(feature_dir, filename)
+        if f.endswith(".parquet"):
+            # for simple features that we want to aggregate, pandas should work fine
+            feature_df = pd.read_parquet(f)
```

Comment:
I'm not sure if this would run out of memory for large datasets, chunking may be advisable. don't necessarily need to correct now but just a note